packages/grafana-ui/src/components/SecretFormField/SecretFormField.tsx

```diff
@@ -77,6 +77,7 @@ export const SecretFormField: FunctionComponent<Props> = ({
             className={`gf-form-input width-${inputWidth}`}
             placeholder={placeholder}
             {...inputProps}
+            disabled={inputProps.disabled || inputProps.readOnly}
```

Comment:
As previous comment can't we continue to use `disabled` here?

Reply:
fixed